web/pdf_find_controller.js

```diff
@@ -120,16 +182,24 @@ var PDFFindController = (function PDFFindControllerClosure() {
         query = query.toLowerCase();
       }
 
-      var matches = [];
-      var matchIdx = -queryLen;
-      while (true) {
-        matchIdx = pageContent.indexOf(query, matchIdx + queryLen);
-        if (matchIdx === -1) {
-          break;
-        }
-        matches.push(matchIdx);
+      if (phraseSearch) {
+        addMatches(query);
+      } else {
+        matchesWithLength = [];
+        var queryArray = query.split(DIVIDER_TERMS);
+        queryArray.forEach(addMatches);
       }
+
+
+      if (!phraseSearch) {
```

Comment:
This `if` block can now removed, and its code can instead be placed inside of the `else` block just above.